modules/effects/src/effects_error_handler.ts

```diff
@@ -0,0 +1,25 @@
+import { ErrorHandler } from '@angular/core';
+import { Action } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+export type EffectsErrorHandler = <T extends Action>(
+  observable$: Observable<T>,
+  errorHandler: ErrorHandler
+) => Observable<T>;
+
+// this is the default error handler provided in EffectsModule.forRoot
```

Comment:
Nit, I think this comment can be left out

Reply:
dropped